doc/source/changelog.md

```diff
@@ -4,6 +4,10 @@
 
 ### What's new?
 
+- **Deprecate Python 3.7**
+
+  Since Python 3.7 reached its end of life (EOL), support for Python 3.7 is now deprecated and will be removed in the next release.
```

Comment:
```suggestion
  Since Python 3.7 reached its end of life (EOL) on 2023-06-27, support for Python 3.7 is now deprecated and will be removed in an upcoming release.
```